src/employeemanagement/EmployeeManagement.java

```diff
@@ -0,0 +1,95 @@
+package employeemanagement;
+import java.util.ArrayList;
+import java.util.Scanner;
+public class EmployeeManagement {
+    public static void main(String[] args) {
+        ArrayList<Employee> employeeList = new ArrayList<>();
+        ArrayList<Officer> officerList = new ArrayList<>();
+        int choose;
+        do {
+            Scanner input = new Scanner(System.in);
+            showMenu();
+            System.out.println("Enter Choose Of You: ");
+            choose = Integer.parseInt(input.nextLine());
+            switch (choose){
+                case 1:
+                    int n;
+                    System.out.println("Enter Number Employee Need Add");
+                    n = Integer.parseInt(input.nextLine());
+                    for (int i=0;i<n;i++){
+                        Employee E = new Employee();
+                        E.inputEmployee();
+
+                        employeeList.add(E);
+                    }
+                    break;
+                case 2:
+                    System.out.println("Enter ID Employee Need Delete: ");
+                    String inputName = input.nextLine();
+                    int count =0;
+                    for(Employee employee : employeeList){
```

Comment:
Ở đây: inputName là 1 String, còn hàm employeeList.contains   nó nhận vào là 1 Object kiểu Employee nên không thể delete được. 
Thay vào đó, E có thể duyệt tất cả các phần tử trong employeeList, lấy tên của mỗi employee và so sánh với inputName nhập từ bàn phím. 
Ví dụ: 
for (Employee employee: employeeList) {
     if (employee.getName().equals(inputName)) {    // Em có thể tìm hiểu hàm equals này trước, buổi sau c sẽ giải thích 
 //kỹ hơn. Hàm equals ở đây tương tự như dấu ==. Tương tự chứ không phải là giống. 
             employeeList.remove(employee) // remove cái employee này bởi vì nó có tên giống tên của inputName nhập từ bàn phím.
}
}